backend/app/models/question.rb

```diff
@@ -1,26 +1,25 @@
 require 'numbers_in_words'
```

Comment:
Btw, you don't need to require `numbers_in_words` library anymore. It should be already loaded, hence usable, once the app has been initialized by Rails.